tls/s2n_resume.h

```diff
@@ -99,3 +104,13 @@ extern int s2n_store_to_cache(struct s2n_connection *conn);
  * once we release the session resumption API. */
 int s2n_config_set_initial_ticket_count(struct s2n_config *config, uint8_t num);
 int s2n_connection_add_new_tickets_to_send(struct s2n_connection *conn, uint8_t num);
+
+struct s2n_session_ticket {
+    struct s2n_blob ticket_data;
+    uint32_t session_lifetime;
+};
```

Comment:
You've got this below the "Will become publicly visible" line in this header. I'd move the struct definition up above it and just put `struct s2n_session_ticket;` down here, to avoid copy-paste errors when you go to release the API and to make it clear that the structure is intended to be opaque.